src/Cake.Common/Tools/NUnit/NUnitRunner.cs

```diff
@@ -45,15 +46,35 @@ public void Run(FilePath assemblyPath, NUnitSettings settings)
                 throw new ArgumentNullException("settings");
             }
 
-            Run(settings, GetArguments(assemblyPath, settings), settings.ToolPath);
+            Run(settings, GetArguments(new [] { assemblyPath }, settings), settings.ToolPath);
         }
 
-        private ProcessArgumentBuilder GetArguments(FilePath assemblyPath, NUnitSettings settings)
+        /// <summary>
+        /// Runs the tests in the specified assemblies.
+        /// </summary>
+        /// <param name="assemblyPaths">The assembly paths.</param>
+        /// <param name="settings">The settings.</param>
+        public void Run(List<FilePath> assemblyPaths, NUnitSettings settings)
```

Comment:
This is very specific. Maybe `List<FilePath` could be replaced with a `IEnumerable<FilePath>`.